myGitApp/app/src/main/java/com/song2/mygitapp/ui/feed/TestListAdapter.kt

```diff
@@ -0,0 +1,47 @@
+package com.song2.mygitapp.ui.feed
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+import com.song2.mygitapp.databinding.ItemFollowersBinding
+
+class TestListAdapter : ListAdapter<Test, TestListAdapter.ViewHolder>(
+    DiffTool()
+) {
+
+    private class DiffTool : DiffUtil.ItemCallback<Test>(){
+        override fun areItemsTheSame(oldItem: Test, newItem: Test): Boolean {
+            //아이디가 같은 경우
+            return oldItem.id == newItem.id
+        }
+
+        override fun areContentsTheSame(oldItem: Test, newItem: Test): Boolean {
+            //내용이 같은 경우
+            return oldItem == newItem
+        }
+    }
+
+    inner class ViewHolder(val binding: ItemFollowersBinding) : RecyclerView.ViewHolder(binding.root) {
```

Comment:
```suggestion
    inner class ViewHolder(private val binding: ItemFollowersBinding) : RecyclerView.ViewHolder(binding.root) {
```
여기서만 사용하는 변수는 private를 붙여주는게 좋아

Reply:
스터디때 따라 치던 코드...! ㅋㅋ..ㅠㅠ 새로 적용한 코드에는 모두 적용했습니다~